maintenance_checklist_timer/models/maintenance_checklist.py

```diff
@@ -0,0 +1,52 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import _, models
+
+
+class MaintenanceChecklist(models.Model):
+    _name = "maintenance.checklist"
+    _inherit = ["maintenance.checklist", "timer.mixin"]
```

Comment:
This Mixin belongs to a enterprise module :(
https://github.com/odoo/enterprise/blob/14.0/timer/models/timer_mixin.py